onnxruntime/core/providers/opencl/tensor/resize_image2d.cc

```diff
@@ -0,0 +1,112 @@
+#include "resize.h"
+
+#include "core/providers/cpu/tensor/upsample.h"
+#include "core/providers/opencl/opencl_kernel.h"
+#include "core/providers/opencl/opencl_utils.h"
+
+namespace onnxruntime {
+namespace opencl {
+
+namespace {
+#define CONTENT_NAME resize_kernel_src
+#include "opencl_generated/tensor/kernels/resize_image2d.cl.inc"
+}  // namespace
+
+class Resize : public OpenCLKernel, UpsampleBase {
+ public:
+  explicit Resize(const OpKernelInfo& info)
+      : OpenCLKernel(info), UpsampleBase(info) {
+    VLOGS_DEFAULT(0) << "Init Resize (OpenCLKernel)";
+    LoadProgram(resize_kernel_src, resize_kernel_src_len);
+    LoadKernel("ResizeBilinear2D");
+    LoadKernel("ResizeNearest2D");
```

Comment:
Can you selectively load just one kernel by checking the attributes in OpKernelInfo?

Reply:
Done